### DISCUSSION TITLE
Running DIA-NN on windows 11

### DISCUSSION THREAD

**Discussion by pejota66** [vdemichev/DiaNN#731] [General]
Hi,

would it be possible to run DIA-NN on windows 11 in the near future?

Thanks.

**Comment by vdemichev** [comment]
Should be

**Reply by pejota66** [reply]
Hello Vadim,

sorry for bothering you with this again. Do you have any idea when would be possible to run DIA-NN on windows 11? Thank you

**Reply by vdemichev** [reply]
It works on Windows 11 just fine

**Reply by pejota66** [reply]
Since Monday, I have been able to run DIANN on Windows 11.

**Comment by pejota66** [comment]
I am unable to run on Windows 11. Dia-NN starts and displays the initializing library code, but it crashes when loading the first raw file.

[0:19] Initialising library

[0:54] File #1/6
[0:54] Loading run C:\Users\u0161226\Desktop\K562vsHELA_Swathand30SPD\HELA1.wiff

DIA-NN exited

DIA-NN-plotter.exe "C:\GBW_MyPrograms\DIA-NN\report.stats.tsv" "C:\GBW_MyPrograms\DIA-NN\report.tsv" "C:\GBW_MyPrograms\DIA-NN\report.pdf"
PDF report will be generated in the background

**Reply by vdemichev** [reply]
Please copy Sciex .dll files to DIA-NN folder as described in the docs

**Reply by pejota66** [reply]
I have done this before, without success but now is up and running. Thanks Vadim Demichev!

**Comment by AshokPrabhu123** [comment]
Hi all,
I am getting the same error too. Kindly provide a solution.
Regards, 
Ashok

**Reply by vdemichev** [reply]
Please see above
